Lib/extractor/formats/type1.py

```diff
@@ -28,7 +28,12 @@ def extractFontFromType1(
     doKerning=True,
     customFunctions=[],
 ):
-    source = T1Font(pathOrFile)
+    try:
+        source = T1Font(pathOrFile, encoding="ascii")
+        source["FontInfo"]
+    except UnicodeDecodeError:
+        source = T1Font(pathOrFile, encoding="macroman")
```

Comment:
What do you gain by first trying with ascii? It is a subset of macroman anyway, so there's zero difference in behavior with just using macroman straight away.

Reply:
true, fixed "macroman" it is